### DISCUSSION TITLE
Error when using q-scroll-observer @scroll event with typescript

### DISCUSSION THREAD

**Discussion by tgarif** [quasarframework/quasar#10182] [General - Components / Directives / etc]
Hi guys, i was using q-scroll-observer that emits @scroll function with typescript. I created a onScroll function to get the default 'details' params from the @scroll event emitted but it says 'details' implicitly has 'any' type...

![Screenshot (45)](https://user-images.githubusercontent.com/60260466/126892581-89b8c077-3621-484f-b4d3-63a45c2c582a.png)

Anyone knows how to solve this?

**Comment by MilosPaunovic** [comment]
```js
const onScroll = ((details: any) : { position: { top: number } }) => {
  // other stuff
}
```
